clusters/app.ci/assets/admin_qci-appci.yaml

```diff
@@ -93,6 +93,18 @@ spec:
       labels:
         app: qci-appci
     spec:
+      affinity:
```

Comment:
Might you explain a bit the rationale behind this decision?

Reply:
This is to run the two replicas on different nodes to ensure the HA, e.g., while the node gets drained.